### PR TITLE
teams/migration: a vote for D3Hunter and WizardXiao as the reviewer

### PR DESCRIPTION
**Pull request by glorv**
RT

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]
LGTM. Congratulations!

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by Ehco1996 — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]
also need to change https://github.com/pingcap/community/blob/c842008b9d0a5a3d06015dcdf7673d2941343a53/teams/migration/membership.json#L37

**Comment by glorv** [comment]
> also need to change
> 
> https://github.com/pingcap/community/blob/c842008b9d0a5a3d06015dcdf7673d2941343a53/teams/migration/membership.json#L37

done 😅

**Comment by glorv** [comment]
/hold 
Wait review comment for 3 days.

File: votes/0619-D3Hunter-WizardXiao-as-migration-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# A Vote for D3Hunter and WizardXiao as Migration Reviewer
+
+## Proposal
+
+[@D3Hunter](https://github.com/D3Hunter) has been working on `pingcap/tiflow` for several months, he has made significant contributions to DM's relay log module performance optimization:
+
+* [Authored pull requests(tiflow)](https://github.com/pingcap/tiflow/pulls?q=is%3Apr+author%3AD3Hunter+is%3Aclosed).
+* [Authored pull requests(dm)](https://github.com/pingcap/dm/pulls?q=is%3Apr+author%3AD3Hunter+is%3Aclosed).
+
+[@WizardXiao](https://github.com/WizardXiao) has been working on `pingcap/tiflow` for several months, he has made  contributions in DM's sync module.
```
Comment (amyangfei) [inline votes/0619-D3Hunter-WizardXiao-as-migration-reviewer.md:10]:
```suggestion
[@WizardXiao](https://github.com/WizardXiao) has been working on `pingcap/tiflow` for several months, he has made contributions in DM's sync module.
```

File: teams/migration/membership.json [hunk 2]
```diff
@@ -38,11 +38,13 @@
...
         "asddongmen",
         "ben1009",
+        "D3Hunter",
         "gozssky",
         "iamxy",
         "okJiang",
         "sdojjy",
-        "tiancaiamao"
+        "tiancaiamao",
+        "WizardXiao"
```
Comment (amyangfei) [inline teams/migration/membership.json:47]: Seems the name is sorted by ascii order, but I found `hi-rustin` is out of order😂

**Review by amyangfei — COMMENTED** [review]
the vote result is LGTM.
Congrats!

**Review by amyangfei — APPROVED** [review]

**Pull request merged**